jupyterhub/app.py

```diff
@@ -2291,7 +2291,10 @@ def ask():
 
         # start the webserver
         self.http_server = tornado.httpserver.HTTPServer(
-            self.tornado_application, ssl_options=ssl_context, xheaders=True
+            self.tornado_application,
+            ssl_options=ssl_context,
+            xheaders=True,
+            trusted_downstream=['127.0.0.1'],
```

Comment:
The http proxy may not be on localhost. Can you add this as a `trusted_downstream_ips = List(Unicode(), default_value=['127.0.0.1'], config=True, help="""explanation...""")` trait so that it can be set when the proxy might be remote?